dockers/piped-base/DOCKER_BUILD

```diff
@@ -1,2 +1,2 @@
-version: 0.1.0
+version: 0.1.1
```

Comment:
I'm not sure why do we need to update this?

Reply:
👍